packages/agate/template/src/index.js

```diff
@@ -9,3 +10,8 @@ if (typeof window !== 'undefined') {
 }
 
 export default appElement;
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
```

Comment:
should we keep this comment on?
this comment & link is from CRA.

Reply:
I think we can keep the description.
please decide when you merge. @MikyungKim

Reply:
I think it would be better if we create our own doc instead of linking CRA's.